modules/exploits/linux/http/freepbx_hotelwakeup_rce.rb

```diff
@@ -0,0 +1,120 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit4 < Msf::Exploit::Remote
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+                      'Name'            => 'FreePBX < 13.0.188.1 Remote root exploit',
+                      'Description'     => '
+                        This module exploits an unauthenticated remote command execution in FreePBX module Hotelwakeup
+                      ',
+                      'License'         => MSF_LICENSE,
+                      'Author'          =>
+                        [
+                          'Ahmed sultan (0x4148) <0x4148@gmail.com>', # discovery of vulnerability and msf module
+                        ],
+                      'References'      =>
+                        [
+                          "NA"
```

Comment:
```['EDB', '40434'],
['URL', 'http://wiki.freepbx.org/display/FOP/2016-08-09+CVE+Remote+Command+Execution+with+Privileged+Escalation']```
possibly 40614, although I didn't look close at the code to see if it was calling the same things, plus this module came out first